src/pages/josungmin/Login/Login.scss

```
@@ -0,0 +1,99 @@
+.container {
+    width: 350px;
+    height: 380px;
+    margin: 100px auto;
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    flex-wrap: wrap;
+    background-color: #fff;
+    border: 1px solid #DBDBDB;
+}
+.westaLogo {
+    font-family: 'Lobster', cursive;
+    font-size: 50px;
+    font-weight: 500;
+    margin-top: 30px;
+    margin-bottom: 30px;
```

```suggestion
    margin-top: 30px;
    margin-bottom: 30px;
    font-family: 'Lobster', cursive;
    font-size: 50px;
    font-weight: 500
```
css convention에 맞춰 순서를 바꿔주시면 좋을 것 같습니다!
추가로 font-weight는 400  = normal, 700 = bold이니 참고해주세요!